boards.txt

```diff
@@ -1556,17 +1567,6 @@ GenF4.menu.xusb.HS.build.usb_speed=-DUSE_USB_HS
 GenF4.menu.xusb.HSFS=High Speed in Full Speed mode
 GenF4.menu.xusb.HSFS.build.usb_speed=-DUSE_USB_HS -DUSE_USB_HS_IN_FS
 
-Sparky.menu.usb.none=None
```

Comment:
I think it should be better to keep USB menu.
Even if the wraith32 does not have USB, the  B-G431B-ESC1 have it.

Reply:
Not done.  USB on B-G431B-ESC1 is part of ST-Link breakoff board.  I know of no Electronic Speed Control with USB connectivity.  In general ESC communication and programming is done via serial connection to a flight controller, which does have a USB connection to a PC.

Reply:
OK